### ISSUE TITLE
Build a front end dashboard to Visualize creek scores and other data

### ISSUE THREAD

**Issue by mateoclarke** [open-austin/water-quality#4]

**Comment by mateoclarke** [comment]
Hey @sanjitpal09, let's talk about the idea of making some visualizations with Tableau. I don't have much experience with it, but I've heard its an awesome tool. What questions do you have to get started?

**Comment by sanjitpal09** [comment]
Where is the data available and in what form?

**Comment by mateoclarke** [comment]
City of Austin Water Quality Sampling Data is the best source I've found but there are many more.
https://data.austintexas.gov/Environmental/Water-Quality-Sampling-Data/5tye-7ray

You can preview what gets published, download the data as a static file in many formats, or hook into the Socrata API.
![image](https://cloud.githubusercontent.com/assets/5697474/26813605/f06effec-4a44-11e7-81af-7b1514016428.png)

**Comment by sanjitpal09** [comment]
Thanks I'll go over the data asap.

**Comment by mudspringhiker** [comment]
Is that Sanjit from a hackathon I went to a couple of weeks ago? Sorry for the long absence from Open Austin.  I have kind of been working (on and off) on this, though it would help if someone can help. I started to create a web app that looks at E. coli (need more work) only. The codes are updated in my github: https://github.com/mudspringhiker/water_quality_austin. (see the flask folder)

I actually was able to talk to the person who provides the data. I just need to followup. 

I can't attend next Wednesday, but maybe the next Saturday one but I will try to contact the water quality guy I previously talked to now that I kind of have an idea how to go about the app.

Thanks for understanding!

**Comment by mudspringhiker** [comment]
You can download the flask folder and try to run the python program to create a local copy of the web app.

**Comment by sanjitpal09** [comment]
Yes I'm that same guy. @mudspringhiker Are you using the same CSV as provided by @mateoclarke or a different one as you are talking about a person from whom you would get the data. I just read about Bokeh and it's use in making dashboards using flask. Maybe I can try that too.

**Comment by mudspringhiker** [comment]
The data I used for the data exploration in the ipython notebook is from
January, I think. But the data being scraped by the flask app is whatever's
on the website. That's great, Sanjit. You can also go ahead and improve the
flask app. Thanks!

**Comment by mudspringhiker** [comment]
Hello again! I have been out but I learned about Flask and made a simple web app using it for the 2017 E. coli data scraped (using the data.austintexas.gov API). Still an initial web app (just a fun project I did after learning something new about Flask). Here is the link (it's deployed in Heroku): https://immense-hamlet-63542.herokuapp.com/
